modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/inputs/SXPBlueprintSelectorInput.js

```diff
@@ -94,14 +95,14 @@ function SXPBlueprintSelectorInput({
 						loading: false,
 						title:
 							!ok || data.status === 'NOT_FOUND'
-								? JSON.stringify(sxpBlueprintId)
+								? sxpBlueprintExternalReferenceCode
 								: data.title,
 					});
 				})
 				.catch(() => {
 					setSXPBlueprint({
 						loading: false,
-						title: JSON.stringify(sxpBlueprintId),
+						title: sxpBlueprintExternalReferenceCode,
```

Comment:
My SF does this:

	selectedExternalReferenceCode={
		sxpBlueprintExternalReferenceCode || ''
	}